cameras:
- camera:
    distortion:
      parameters:
        cols: 1
        rows: 4
        data: [-0.015406579276683843, -0.002083501696756997, -0.0008278623630992491,
          0.00016705459378783482]
      type: equidistant
    image_height: 1024
    image_width: 1280
    intrinsics:
      cols: 1
      rows: 4
      data: [606.9253030992462, 606.6785317789071, 642.1596590069145, 496.93019970438536]
    label: cam0
    line-delay-nanoseconds: 0
    type: pinhole
  T_B_C:
    cols: 4
    rows: 4
    data: [0.9999972985291825, 0.0023071224395196876, 0.0002830554453912646, -0.09994371094029525,
      0.0023009385359458153, -0.9997935845077203, 0.020186482087204537, -0.019314814335435148,
      0.0003295697041605137, -0.020185776260830172, -0.9997961921414568, -0.010213677010282593,
      0.0, 0.0, 0.0, 1.0]
  serial_no: 0
  calib_date: 0
  description: /sync/cam0/image_raw
- camera:
    distortion:
      parameters:
        cols: 1
        rows: 4
        data: [-0.016382783469615594, -0.0017149637104655997, -0.0003481494373303247,
          -4.407567194315975e-05]
      type: equidistant
    image_height: 1024
    image_width: 1280
    intrinsics:
      cols: 1
      rows: 4
      data: [605.5573160128891, 605.430071140307, 634.0044415299614, 516.0185143828766]
    label: cam1
    line-delay-nanoseconds: 0
    type: pinhole
  T_B_C:
    cols: 4
    rows: 4
    data: [0.9999311000827407, 0.009958406828736955, -0.0062149192072868086, 0.09971618486622726,
      0.010124116808686082, -0.9995780018330394, 0.027227201661425157, -0.017556929987488475,
      -0.005941156971820992, -0.027288246277494602, -0.9996099510653815, -0.010692131105538655,
      0.0, 0.0, 0.0, 1.0]
  serial_no: 0
  calib_date: 0
  description: /sync/cam1/image_raw
label: camchain-imucam-cam_imu.yaml

imu_params:
  delay_imu_cam: 0.0
  max_imu_delta_t: 0.01
  acc_max: 176.0
  omega_max: 7.8
  sigma_omega_c: 0.01
  sigma_acc_c: 0.02
  sigma_omega_bias_c: 8.0e-06
  sigma_acc_bias_c: 0.0004
  sigma_integration: 0.0
  g: 9.8082
  imu_rate: 200.0

imu_initialization:
  velocity: [0.0, 0.0, 0.0]
  omega_bias: [0.0, 0.0, 0.0]
  acc_bias: [0.0, 0.0, 0.0]
  velocity_sigma: 1.0
  omega_bias_sigma: 0.01
  acc_bias_sigma: 0.02